integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -258,6 +258,8 @@ else if (type instanceof DecimalType) {
         return new BigDecimal(new BigInteger(String.valueOf(rawdata)),
             ((DecimalType) type).getScale());
       }
+    } else if (type.equals(TimestampType.TIMESTAMP)) {
```

Comment:
You can use `type == DataTypes.TIMESTAMP` instead

Reply:
Here I am comparing the type with Presto's datatypes that's why I have used `TimestampType.TIMESTAMP`